experiments/simbricks/orchestration/experiment/experiment_environment.py

```diff
@@ -66,11 +66,19 @@ def gem5_path(self, variant: str) -> str:
     def hdcopy_path(self, sim: 'simulators.Simulator') -> str:
         return f'{self.workdir}/hdcopy.{sim.name}'
 
-    def hd_path(self, hd_name: str) -> str:
-        return f'{self.repodir}/images/output-{hd_name}/{hd_name}'
-
-    def hd_raw_path(self, hd_name: str) -> str:
-        return f'{self.repodir}/images/output-{hd_name}/{hd_name}.raw'
+    @staticmethod
+    def is_absolute_exists(path: str) -> bool:
+        return os.path.isabs(path) and os.path.exists(path) and os.path.isfile(path)
+
+    def hd_path(self, hd_name_or_path: str) -> str:
+        if ExpEnv.is_absolute_exists(hd_name_or_path):
+            return hd_name_or_path
+        return f'{self.repodir}/images/output-{hd_name_or_path}/{hd_name_or_path}'
+
+    def hd_raw_path(self, hd_name_or_path: str) -> str:
+        if ExpEnv.is_absolute_exists(hd_name_or_path):
+            return hd_name_or_path
```

Comment:
That's actually a good point. Wasn't sure how to go about this. I for now thought it is nicer to require users to actually specify that themselves.

Reply:
I'm for adding the .raw file extension here. Otherwise, you have to manually do this in the experiment script via case distinction. This is not something that users should have to do.